components/Common/ShellBox/__tests__/index.test.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import userEvent from '@testing-library/user-event';
+import { render, screen } from '@testing-library/react';
+
+import ShellBox from '../index';
+
+Object.assign(navigator, {
```

Comment:
Instead of assigning this here, you can create a mock on the beforeEach of each test and restore the navigator to its original value on the afterEach. The reasoning why is that depending on the Jest installation, this could interfere other test suites (it shouldn't by default) but it's also a good design to not assign/modify global variables on the main thread.